server/lib/tuist/billing.ex

```diff
@@ -30,6 +30,17 @@ defmodule Tuist.Billing do
     @unit_prices
   end
 
+  @doc """
+  Gets month-to-date remote cache hits count for an account.
+  Returns an integer, defaulting to 0 when data is missing.
+  """
+  def month_to_date_remote_cache_hits_count(account_id) when is_integer(account_id) do
```

Comment:
```suggestion
  def month_to_date_remote_cache_hits_count(account_id) do
```

I think we'd want to crash anyways when the `account_id` is not an integer